rtuclient.go

```diff
@@ -276,19 +285,29 @@ func (mb *rtuSerialTransporter) Send(aduRequest []byte) (aduResponse []byte, err
 	return
 }
 
-// calculateDelay roughly calculates time needed for the next frame.
+// 11bit encoding (https://stackoverflow.com/questions/20740012/calculating-modbus-rtu-3-5-character-time).
 // See MODBUS over Serial Line - Specification and Implementation Guide (page 13).
-func (mb *rtuSerialTransporter) calculateDelay(chars int) time.Duration {
-	var characterDelay, frameDelay int // us
 
+// frameDelay roughly calculates the time needed between two frames.
+func (mb *rtuSerialTransporter) frameDelay() time.Duration {
+	var frameDelay int // us
 	if mb.BaudRate <= 0 || mb.BaudRate > 19200 {
-		characterDelay = 750
 		frameDelay = 1750
 	} else {
-		characterDelay = 15000000 / mb.BaudRate
-		frameDelay = 35000000 / mb.BaudRate
+		frameDelay = 38500000 / mb.BaudRate
+	}
+	return time.Duration(frameDelay) * time.Microsecond
+}
+
+// calculateDelay roughly calculates time needed for the next frame.
+func (mb *rtuSerialTransporter) calculateDelay(chars int) time.Duration {
+	var characterDelay int // us
+	if mb.BaudRate <= 0 || mb.BaudRate > 19200 {
+		characterDelay = 750
+	} else {
+		characterDelay = 16500000 / mb.BaudRate
```

Comment:
```suggestion
		characterDelay = 11000000 / mb.BaudRate
```

The factor 1.5 is wrong here.
T1.5 is the gap timeout on the character stream.
That must be handled inside the stream parser.